sci-misc/netlogo-bin/netlogo-bin-6.2.0.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop java-pkg-2 xdg
+
+MY_PN="NetLogo"
+MY_P=${MY_PN}-${PV}
+
+DESCRIPTION="Cross-platform multi-agent programmable modelling environment"
```

Comment:
Could we find something more descriptive? Probably in a separate PR for all ebuilds. I do not understand what "Cross-platform multi-agent programmable modelling environment" is.

Reply:
Changed it to "A programming language and IDE for agent-based modelling" (which is from Wikipedia) in a second commit, is this more clear?

Reply:
Just a nitpick but generally I try to avoid starting descriptions with "A", i.e. "Programming language and IDE..."